### PR TITLE
Update `podman_image` to remove image with image id

### LINKED ISSUE #357: [`podman_image` Feature Request] Allow removing image with image id
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

I would like to be able to delete an image by its UID using the `podman_image` module.

**Steps to reproduce the issue:**

1. Pull an image

2. Attempt to remove an image using `-m podman_image`, `-a state=absent` and pass a container id instead of an image tag to `name`

**Describe the results you received:**
The ansible task exits with status `OK` and doesn't actually remove the image.

**Describe the results you expected:**

The module should be able to remove an image by its name **or its id**. I marked this as a feature request, because the module does not claim to support setting `name` to an image ID.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version of the `containers.podman` collection:**
**Either git commit if installed from git: `git show --summary`**
**Or version from `ansible-galaxy` if installed from galaxy: `ansible-galaxy collection list | grep containers.podman`**

```
containers.podman 1.9.0
```

**Output of `ansible --version`:**

```
ansible [core 2.12.1]
  config file = /home/gotmax/.ansible.cfg
  configured module search path = ['/home/gotmax/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.10/site-packages/ansible
  ansible collection location = /home/gotmax/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.1 (main, Dec  7 2021, 00:00:00) [GCC 11.2.1 20211203 (Red Hat 11.2.1-7)]
  jinja version = 3.0.1
  libyaml = True
```

**Output of `podman version`:**

```
podman version 3.4.4
```

**Output of `podman info --debug`:**

``` yaml
host:
  arch: amd64
  buildahVersion: 1.23.1
  cgroupControllers:
  - cpu
  - io
  - memory
  - pids
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: conmon-2.0.30-2.fc35.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.0.30, commit: '
  cpus: 12
  distribution:
    distribution: fedora
    variant: workstation
    version: "35"
  eventLogger: journald
  hostname: ftop
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  kernel: 5.15.10-200.fc35.x86_64
  linkmode: dynamic
  logDriver: journald
  memFree: 698245120
  memTotal: 16390434816
  ociRuntime:
    name: crun
    package: crun-1.3-1.fc35.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 1.3
      commit: 8e5757a4e68590326dafe8a8b1b4a584b10a1370
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +YAJL
  os: linux
  remoteSocket:
    exists: true
    path: /run/user/1000/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: true
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns-1.1.12-2.fc35.x86_64
    version: |-
      slirp4netns version 1.1.12
      commit: 7a104a101aa3278a2152351a082a6df71f57c9a3
      libslirp: 4.6.1
      SLIRP_CONFIG_VERSION_MAX: 3
      libseccomp: 2.5.3
  swapFree: 15250890752
  swapTotal: 16454242304
  uptime: 75h 29m 14.19s (Approximately 3.12 days)
plugins:
  log:
  - k8s-file
  - none
  - journald
  network:
  - bridge
  - macvlan
  volume:
  - local
registries:
  search:
  - registry.fedoraproject.org
  - registry.access.redhat.com
  - docker.io
  - quay.io
store:
  configFile: /home/gotmax/.config/containers/storage.conf
  containerStore:
    number: 4
    paused: 0
    running: 3
    stopped: 1
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/gotmax/.local/share/containers/storage
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 58
  runRoot: /run/user/1000/containers
  volumePath: /home/gotmax/.local/share/containers/storage/volumes
version:
  APIVersion: 3.4.4
  Built: 1638999907
  BuiltTime: Wed Dec  8 16:45:07 2021
  GitCommit: ""
  GoVersion: go1.16.8
  OsArch: linux/amd64
  Version: 3.4.4
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-3.4.4-1.fc35.x86_64
```

**Playbok you run with ansible (e.g. content of `playbook.yaml`):**

``` yaml
#!/usr/bin/env -S ansible-playbook
# vim: set filetype=yaml.ansible:
- name: podman_image_update
  hosts: localhost
  gather_facts: false
  connection: local

  tasks:
    - name: List images
      register: podman_images
      changed_when: false
      ansible.builtin.shell:
        cmd: podman images | grep -v '<none>\|localhost' | awk 'NR>1 {print $1 ":" $2}'

    - name: Pull images
      loop: "{{ podman_images['stdout_lines'] }}"
      containers.podman.podman_image:
        name: "{{ item }}"
        pull: true

    - name: List images to prune
      register: podman_images_prune
      changed_when: false
      ansible.builtin.shell:
        cmd: podman images | grep '<none>' | awk '{print $3}'

    - name: Prune images
      loop: "{{ podman_images_prune['stdout_lines'] }}"
      containers.podman.podman_image:
        name: "{{ item }}"
        state: absent

```

**Command line and output of ansible run with high verbosity**

**Please NOTE: if you submit a bug about idempotency, run the playbook with `--diff` option, like:**

`ansible-playbook -i inventory --diff -vv playbook.yml`

```
Using /home/gotmax/.ansible.cfg as config file

PLAY [podman_image_update] *****************************************************

TASK [List images] *************************************************************
ok: [localhost] => changed=false 
  cmd: podman images | grep -v '<none>\|localhost' | awk 'NR>1 {print $1 ":" $2}'
  delta: '0:00:03.052400'
  end: '2021-12-26 12:05:09.827680'
  msg: ''
  rc: 0
  start: '2021-12-26 12:05:06.775280'
  stderr: ''
  stderr_lines: <omitted>
  stdout: |-
    docker.io/library/python:3.9-slim-bullseye
    quay.io/gotmax23/opensuse-systemd:tumbleweed
    quay.io/gotmax23/archlinux-dotfiles:latest
    quay.io/gotmax23/archlinux-systemd:latest
    quay.io/gotmax23/archlinux-systemd:any
    quay.io/gotmax23/debian-systemd:buster
    quay.io/gotmax23/el-systemd:latest
    quay.io/gotmax23/el-systemd:8
    quay.io/gotmax23/debian-systemd:bookworm
    quay.io/gotmax23/ubuntu-systemd:impish
    quay.io/gotmax23/fedora-systemd:33
    quay.io/gotmax23/fedora-systemd:34
    quay.io/gotmax23/ubuntu-systemd:bionic
    quay.io/gotmax23/fedora-systemd:latest
    quay.io/gotmax23/fedora-systemd:36
    quay.io/gotmax23/opensuse-systemd:15.2
    quay.io/gotmax23/ubuntu-systemd:focal
    quay.io/gotmax23/centos-systemd:stream8
    quay.io/gotmax23/redhat-systemd:8
    quay.io/gotmax23/fedora-systemd:35
    quay.io/gotmax23/opensuse-systemd:latest
    quay.io/gotmax23/opensuse-systemd:15.3
    quay.io/gotmax23/centos-systemd:stream9
    quay.io/gotmax23/ubuntu-systemd:22.04
    quay.io/gotmax23/el-systemd:7
    quay.io/gotmax23/debian-systemd:latest
    quay.io/gotmax23/debian-systemd:bullseye
    quay.io/gotmax23/debian-systemd:11
    quay.io/gotmax23/ubuntu-systemd:hirsute
    quay.io/gotmax23/archlinux-dotfiles:local
    quay.io/centos/centos:stream9
    registry.access.redhat.com/ubi8:latest
    registry.fedoraproject.org/fedora:36
    docker.io/robertdebock/alpine:latest
    docker.io/library/alpine:latest
    docker.io/almalinux/8-base:8
    registry.fedoraproject.org/fedora-toolbox:35
    registry.fedoraproject.org/fedora-toolbox:36
    quay.io/almalinux/8-base:latest
    quay.io/centos/centos:7
  stdout_lines: <omitted>

TASK [Pull images] *************************************************************
ok: [localhost] => (item=docker.io/library/python:3.9-slim-bullseye) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: docker.io/library/python:3.9-slim-bullseye
  podman_actions:
  - /usr/bin/podman image ls docker.io/library/python:3.9-slim-bullseye --format json
ok: [localhost] => (item=quay.io/gotmax23/opensuse-systemd:tumbleweed) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/opensuse-systemd:tumbleweed
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/opensuse-systemd:tumbleweed --format json
ok: [localhost] => (item=quay.io/gotmax23/archlinux-dotfiles:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/archlinux-dotfiles:latest
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/archlinux-dotfiles:latest --format json
ok: [localhost] => (item=quay.io/gotmax23/archlinux-systemd:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/archlinux-systemd:latest
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/archlinux-systemd:latest --format json
ok: [localhost] => (item=quay.io/gotmax23/archlinux-systemd:any) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/archlinux-systemd:any
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/archlinux-systemd:any --format json
ok: [localhost] => (item=quay.io/gotmax23/debian-systemd:buster) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/debian-systemd:buster
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/debian-systemd:buster --format json
ok: [localhost] => (item=quay.io/gotmax23/el-systemd:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/el-systemd:latest
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/el-systemd:latest --format json
ok: [localhost] => (item=quay.io/gotmax23/el-systemd:8) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/el-systemd:8
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/el-systemd:8 --format json
ok: [localhost] => (item=quay.io/gotmax23/debian-systemd:bookworm) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/debian-systemd:bookworm
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/debian-systemd:bookworm --format json
ok: [localhost] => (item=quay.io/gotmax23/ubuntu-systemd:impish) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/ubuntu-systemd:impish
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/ubuntu-systemd:impish --format json
ok: [localhost] => (item=quay.io/gotmax23/fedora-systemd:33) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/fedora-systemd:33
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/fedora-systemd:33 --format json
ok: [localhost] => (item=quay.io/gotmax23/fedora-systemd:34) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/fedora-systemd:34
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/fedora-systemd:34 --format json
ok: [localhost] => (item=quay.io/gotmax23/ubuntu-systemd:bionic) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/ubuntu-systemd:bionic
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/ubuntu-systemd:bionic --format json
ok: [localhost] => (item=quay.io/gotmax23/fedora-systemd:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/fedora-systemd:latest
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/fedora-systemd:latest --format json
ok: [localhost] => (item=quay.io/gotmax23/fedora-systemd:36) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/fedora-systemd:36
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/fedora-systemd:36 --format json
ok: [localhost] => (item=quay.io/gotmax23/opensuse-systemd:15.2) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/opensuse-systemd:15.2
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/opensuse-systemd:15.2 --format json
ok: [localhost] => (item=quay.io/gotmax23/ubuntu-systemd:focal) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/ubuntu-systemd:focal
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/ubuntu-systemd:focal --format json
ok: [localhost] => (item=quay.io/gotmax23/centos-systemd:stream8) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/centos-systemd:stream8
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/centos-systemd:stream8 --format json
ok: [localhost] => (item=quay.io/gotmax23/redhat-systemd:8) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/redhat-systemd:8
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/redhat-systemd:8 --format json
ok: [localhost] => (item=quay.io/gotmax23/fedora-systemd:35) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/fedora-systemd:35
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/fedora-systemd:35 --format json
ok: [localhost] => (item=quay.io/gotmax23/opensuse-systemd:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/opensuse-systemd:latest
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/opensuse-systemd:latest --format json
ok: [localhost] => (item=quay.io/gotmax23/opensuse-systemd:15.3) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/opensuse-systemd:15.3
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/opensuse-systemd:15.3 --format json
ok: [localhost] => (item=quay.io/gotmax23/centos-systemd:stream9) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/centos-systemd:stream9
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/centos-systemd:stream9 --format json
ok: [localhost] => (item=quay.io/gotmax23/ubuntu-systemd:22.04) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/ubuntu-systemd:22.04
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/ubuntu-systemd:22.04 --format json
ok: [localhost] => (item=quay.io/gotmax23/el-systemd:7) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/el-systemd:7
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/el-systemd:7 --format json
ok: [localhost] => (item=quay.io/gotmax23/debian-systemd:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/debian-systemd:latest
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/debian-systemd:latest --format json
ok: [localhost] => (item=quay.io/gotmax23/debian-systemd:bullseye) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/debian-systemd:bullseye
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/debian-systemd:bullseye --format json
ok: [localhost] => (item=quay.io/gotmax23/debian-systemd:11) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/debian-systemd:11
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/debian-systemd:11 --format json
ok: [localhost] => (item=quay.io/gotmax23/ubuntu-systemd:hirsute) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/ubuntu-systemd:hirsute
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/ubuntu-systemd:hirsute --format json
ok: [localhost] => (item=quay.io/gotmax23/archlinux-dotfiles:local) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/gotmax23/archlinux-dotfiles:local
  podman_actions:
  - /usr/bin/podman image ls quay.io/gotmax23/archlinux-dotfiles:local --format json
ok: [localhost] => (item=quay.io/centos/centos:stream9) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/centos/centos:stream9
  podman_actions:
  - /usr/bin/podman image ls quay.io/centos/centos:stream9 --format json
ok: [localhost] => (item=registry.access.redhat.com/ubi8:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: registry.access.redhat.com/ubi8:latest
  podman_actions:
  - /usr/bin/podman image ls registry.access.redhat.com/ubi8:latest --format json
ok: [localhost] => (item=registry.fedoraproject.org/fedora:36) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: registry.fedoraproject.org/fedora:36
  podman_actions:
  - /usr/bin/podman image ls registry.fedoraproject.org/fedora:36 --format json
ok: [localhost] => (item=docker.io/robertdebock/alpine:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: docker.io/robertdebock/alpine:latest
  podman_actions:
  - /usr/bin/podman image ls docker.io/robertdebock/alpine:latest --format json
ok: [localhost] => (item=docker.io/library/alpine:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: docker.io/library/alpine:latest
  podman_actions:
  - /usr/bin/podman image ls docker.io/library/alpine:latest --format json
ok: [localhost] => (item=docker.io/almalinux/8-base:8) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: docker.io/almalinux/8-base:8
  podman_actions:
  - /usr/bin/podman image ls docker.io/almalinux/8-base:8 --format json
ok: [localhost] => (item=registry.fedoraproject.org/fedora-toolbox:35) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: registry.fedoraproject.org/fedora-toolbox:35
  podman_actions:
  - /usr/bin/podman image ls registry.fedoraproject.org/fedora-toolbox:35 --format json
ok: [localhost] => (item=registry.fedoraproject.org/fedora-toolbox:36) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: registry.fedoraproject.org/fedora-toolbox:36
  podman_actions:
  - /usr/bin/podman image ls registry.fedoraproject.org/fedora-toolbox:36 --format json
ok: [localhost] => (item=quay.io/almalinux/8-base:latest) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/almalinux/8-base:latest
  podman_actions:
  - /usr/bin/podman image ls quay.io/almalinux/8-base:latest --format json
ok: [localhost] => (item=quay.io/centos/centos:7) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: quay.io/centos/centos:7
  podman_actions:
  - /usr/bin/podman image ls quay.io/centos/centos:7 --format json

TASK [List images to prune] ****************************************************
ok: [localhost] => changed=false 
  cmd: podman images | grep '<none>' | awk '{print $3}'
  delta: '0:00:03.157861'
  end: '2021-12-26 12:05:27.449711'
  msg: ''
  rc: 0
  start: '2021-12-26 12:05:24.291850'
  stderr: ''
  stderr_lines: <omitted>
  stdout: |-
    a6f8b83c50ae
    b2f753f42c02
    5387d944d710
  stdout_lines: <omitted>

TASK [Prune images] ************************************************************
ok: [localhost] => (item=a6f8b83c50ae) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: a6f8b83c50ae
  podman_actions:
  - /usr/bin/podman image ls a6f8b83c50ae:None --format json
ok: [localhost] => (item=b2f753f42c02) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: b2f753f42c02
  podman_actions:
  - /usr/bin/podman image ls b2f753f42c02:None --format json
ok: [localhost] => (item=5387d944d710) => changed=false 
  actions: []
  ansible_loop_var: item
  image: {}
  item: 5387d944d710
  podman_actions:
  - /usr/bin/podman image ls 5387d944d710:None --format json

PLAY RECAP *********************************************************************
localhost                  : ok=4    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

### PR DESCRIPTION
**Pull request by nishipy**
Fixes #357.

### CODE REVIEW

**Comment by sshnaidm** [comment]
Thanks, can you please add a test here? https://github.com/containers/ansible-podman-collections/blob/master/tests/integration/targets/podman_image/tasks/main.yml
You can use `podman_image_info` module to discover an ID

**Comment by nishipy** [comment]
Thanks for your comment! I added some lines in [tests/integration/targets/podman_image/tasks/main.yml](https://github.com/containers/ansible-podman-collections/blob/master/tests/integration/targets/podman_image/tasks/main.yml).

File: tests/integration/targets/podman_image/tasks/main.yml [hunk 1]
```diff
@@ -79,6 +86,18 @@
         state: absent
       register: rmi5
 
+    - name: Get image id of the target image
+      containers.podman.podman_info:
+        name: docker.io/library/ubuntu
+      register: imageinfo
+
+    - name: Remove an image with image id
+      dev.podman.podman_image:
```
Comment (sshnaidm) [inline tests/integration/targets/podman_image/tasks/main.yml:95]:
```suggestion
      containers.podman.podman_image:
```

**Review by sshnaidm — CHANGES_REQUESTED** [review]

File: tests/integration/targets/podman_image/tasks/main.yml (hunk 1 above)
Comment (sshnaidm) [reply tests/integration/targets/podman_image/tasks/main.yml:95]: Seems like debug leftovers?

Comment (nishipy) [reply tests/integration/targets/podman_image/tasks/main.yml]: Yes, you are right. Let me fix it.

**Comment by sshnaidm** [comment]
Please sign your commit: `git commit -s`, it's must in all repos in this org.
You can do `git commit -s --amend` and `git push --force` just to add the signature.

File: tests/integration/targets/podman_image/tasks/main.yml [hunk 2]
```diff
@@ -79,6 +86,18 @@
         state: absent
       register: rmi5
 
+    - name: Get image id of the target image
+      containers.podman.podman_info:
```
Comment (sshnaidm) [inline tests/integration/targets/podman_image/tasks/main.yml:90]: This should be `containers.podman.podman_image_info`
```suggestion
      containers.podman.podman_image_info:
```

**Review by sshnaidm — CHANGES_REQUESTED** [review]

File: tests/integration/targets/podman_image/tasks/main.yml (hunk 2 above)
Comment (nishipy) [reply tests/integration/targets/podman_image/tasks/main.yml]: Thanks. Fixed it on this commit: 75c5fa1f467abbb91fac341a0f4c218444720eba.

**Comment by nishipy** [comment]
> Please sign your commit: `git commit -s`, it's must in all repos in this org. You can do `git commit -s --amend` and `git push --force` just to add the signature. All commits should be signed.

I added them to all commits so far.

**Pull request merged**